### DISCUSSION TITLE
Option to disable scrolling to the placed object in the editor

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#22667] [Ideas] [answered]
When mapping mania and taiko, the editor scrolling to the object you just placed down if its not at the current time gets _very annoying_, atleast if you're used to stable, quaver, arrowvortex, etc editors.

An option to disable this would be highly appreciated

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/17424
